docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -903,25 +925,86 @@ BUILD(FilterDsLib1);
     without decompression. The original index compression implementation
     decompresses the rows when they are read from disk.</para>
 
+    <para>The inplace index compression format (introduced in versions 9.6.90,
```

Comment:
It isn't the inplace index compression that was introduced  it was already supported.  I am not sure that either of these paragraphs are needed.